build.sh

```diff
@@ -34,5 +34,7 @@ echo installing dependencies from requirements.txt
 $VIRTUAL_ENV/bin/pip install -r requirements.txt -U
 source $VIRTUAL_ENV/bin/activate
 $PYTHON -m PyInstaller --onefile --hidden-import="googleapiclient" --add-data="./src:src"  src/main.py
+# When running as a local module, we need meta.json to be in the same directory as the module.
+ln -sf ../meta.json dist
```

Comment:
This line looks wrong, but it's right. It looks like "look one directory up for a file, and add a symlink in dist/" but it's actually "add to dist/ a symlink that goes one directory up from there (to the current directory).